packages/discovery-provider/src/api/v1/tracks.py

```diff
@@ -192,20 +163,13 @@ class FullTrack(Resource):
             "track_id": "A Track ID",
         },
     )
-    @full_ns.expect(full_track_parser)
+    @full_ns.expect(current_user_parser)
```

Comment:
I feel like this should have its own parser

Reply:
what for?

Reply:
if we were to invest in the python codebase, i would advocate for every API endpoint to have its own parser & marshaller defined in the same file the endpoint is defined in and have that be the only thing in the file. It allows for future modification way easier.
https://overreacted.io/the-wet-codebase/

Reply:
by sharing these parsers, we make it harder to reason about making isolated & safe changes

Reply:
Oh ok, I thought you were saying something was missing from the parser.

I generally agree, although I don't think it's too hard to reason about in this particular scenario? Can just copy the parser and do whatever you need. Current user parser is shared _so much_ and it really shouldn't change at all between usages. I think having every single full endpoint document a current user parameter and try to maintain they're all consistent is probably worse than sharing in this case, but tbh I don't feel strongly.

Probably out of scope for this PR though regardless